nasa-gibs/worldview
# WV-2831: Add contact info to embed page

## Description

Fixes #WV-2831 .
Added a note and mailto link to the embed page asking users to contact us if they want to embed Worldview. 

## How To Test

Check to make sure the embeding.md (https://github.com/nasa-gibs/worldview/blob/wv-2831-add-contact-embed/doc/embeding.md) has the relevant information and the mailto link works. 

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
